lib/ace/commands/default_commands.js

```diff
@@ -622,7 +622,7 @@ exports.commands = [{
     scrollIntoView: "cursor"
 }, {
     name: "transposeletters",
-    bindKey: bindKey("Ctrl-T", "Ctrl-T"),
+    bindKey: bindKey("Alt-X", "Ctrl-T"),
```

Comment:
for multiselect it transposes selected word, which is more useful that teansposing letters